builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -395,6 +396,33 @@ node() {
   }
 }
 
+def attachTagstoLogGroup(config, environment_logical_id, credsId){
+  try {
+    def deploymentDescriptor = null
+    if( env_deployment_descriptor != null ){
+      deploymentDescriptor = env_deployment_descriptor
+    } else {
+      deploymentDescriptor = config['deployment_descriptor']
+    }
+    // If no descriptor present then simply making an empty one. The readYaml default behavior is to return empty string back that is harmful as Map not String is expected below
+    def doc = deploymentDescriptor ? readYaml(text: deploymentDescriptor) : [:]
+    def functionsElem = doc['functions']
+
+    // changing variable names appropriately
+    def serviceDomain = "${configLoader.INSTANCE_PREFIX}_${config['domain']}_${config['service']}"
+    functionsElem.each{functionName, funcConfig ->
+      def lambdaName = funcConfig['name'] ? funcConfig['name'] : "${serviceDomain}_${functionName}_${environment_logical_id}"
```

Comment:
@devsatishm Are these tags for TagGroups sufficient?

Reply:
This is a good start, we can add more when required.